content/company/strategy/index.md

```diff
@@ -59,6 +59,27 @@ Additionally, these are our [product design principles](../../product-engineerin
 - If your organization is getting value from Sourcegraph with a lot of users, our [pricing](https://about.sourcegraph.com/pricing) is designed so that we earn money from you. This lets us invest in improving our product.
 - All users at a given customer are on the same pricing tier. This is simpler than having users at different tiers and encourages us to build things that are broadly valuable.
 
+## Use cases
+
+Use cases help us align the company around the most important value drivers in our product. A good use case should be:
+
+1. TBD
+2. TBD
+3. TBD
+
+Use cases are used by various departments at source graph to do different things:
+
+- [Product](../../product/index.md) (which includes [product management](../../product/product_management/index.md), [design](../../product/design/index.md), and [technical writing](../../product/technical_writing/index.md)) relies on them to help make prioritization decisions because they represent proven value drivers for our customers. The use cases also provide a lens through which the team can evaluate design and workflows within and across product areas, and help they improve outcomes by ensuring that what we build aligns with a coordinated messaging strategy.
```

Comment:
```suggestion
- [Product](../../product-engineering/product/index.md) (which includes [product management](../../product-engineering/product/product_management/index.md), [design](../../product-engineering/product/design/index.md), and [technical writing](../../product-engineering/product/technical_writing/index.md)) relies on them to help make prioritization decisions because they represent proven value drivers for our customers. The use cases also provide a lens through which the team can evaluate design and workflows within and across product areas, and help they improve outcomes by ensuring that what we build aligns with a coordinated messaging strategy.
```